sweetest/src/main/java/com/mysugr/sweetest/framework/coroutine/TestExtensions.kt

```diff
@@ -0,0 +1,28 @@
+package com.mysugr.sweetest.framework.coroutine
+
+import com.mysugr.sweetest.framework.base.BaseJUnitTest
+import com.mysugr.sweetest.framework.base.Steps
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
+
+fun BaseJUnitTest.testCoroutine(
+        testBlock: suspend CoroutineScope.() -> Unit
+) {
+    runBlocking {
+        val coroutinesTestContext = accessor.testContext.coroutines
+        withContext(coroutinesTestContext.coroutineContext) {
+            testBlock()
+        }
+        coroutinesTestContext.testFinished()
+    }
+}
+
+suspend operator fun <T : Steps> T.invoke(run: suspend T.() -> Unit) = run(this)
```

Comment:
i fear, the use of the same operator `invoke` used in the `base` package will cause to confusion some time. also the `base` package will be frequently `*`-imported. i don't know how to handle this best...